build/Microsoft.DotNet.Cli.BranchInfo.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <PropertyGroup>
+    <Channel>master</Channel>
```

Comment:
Shouold we remove BranchInfo.txt and just keep the props file? Where else is that txt being used?